images/cross-cloud/sles-sap/ondemand/profiles.yaml

```diff
@@ -1,14 +1,14 @@
 profiles:
   common:
     include:
-      - base/cloudreg
       - base/common
       - base/sle
       - base/sle-modules
       - products/sap/ondemand
   Azure:
     include:
       - csp/azure/addon/azure-tools
+      - csp/azure/ondemand
```

Comment:
This is superfluous as `csp/gce/ondemand/sap` already includes it. Unless you want it for cosmetic reasons, I suggest removing it.